app/helpers/merge_requests_helper.rb

```diff
@@ -104,10 +104,74 @@ def link_to_merging_organisation_users(organisation)
   end
 
   def total_users_after_merge_text(merge_request)
-    count = merge_request.total_visible_users_after_merge
+    count = merge_request.total_visible_users_after_merge || 0
```

Comment:
Wouldn't total_visible_users_after_merge return 0 if there aren't any anyway?

Reply:
For the stock_owners and managing_agents methods below, in the view nothing/blank would be shown instead of 0. Added it there, added it here too to keep things the similar:
```
def total_stock_owners_after_merge_text(merge_request)
    count = merge_request.total_stock_owners_after_merge || 0
    "#{"#{count} stock owner".pluralize(count)} after merge"
  end

  def total_managing_agents_after_merge_text(merge_request)
    count = merge_request.total_managing_agents_after_merge || 0
    "#{"#{count} managing agent".pluralize(count)} after merge"
  end
```